[2023-12-04T13:07:31.058+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_v0q.first_task scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2023-12-04T13:07:31.075+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_v0q.first_task scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2023-12-04T13:07:31.076+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-04T13:07:31.099+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): first_task> on 2023-12-03 00:00:00+00:00
[2023-12-04T13:07:31.106+0000] {standard_task_runner.py:57} INFO - Started process 62783 to run task
[2023-12-04T13:07:31.113+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_v0q', 'first_task', 'scheduled__2023-12-03T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup.py', '--cfg-path', '/tmp/tmp8w7ci3f1']
[2023-12-04T13:07:31.121+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask first_task
[2023-12-04T13:07:31.245+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_catchup_v0q.first_task scheduled__2023-12-03T00:00:00+00:00 [running]> on host a859eb4bf49d
[2023-12-04T13:07:31.416+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='debora' AIRFLOW_CTX_DAG_ID='dag_with_catchup_v0q' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-03T00:00:00+00:00'
[2023-12-04T13:07:31.419+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-04T13:07:31.421+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hello world, this is my first!']
[2023-12-04T13:07:31.435+0000] {subprocess.py:86} INFO - Output:
[2023-12-04T13:07:31.438+0000] {subprocess.py:93} INFO - hello world, this is my first!
[2023-12-04T13:07:31.440+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-04T13:07:31.490+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_v0q, task_id=first_task, execution_date=20231203T000000, start_date=20231204T130731, end_date=20231204T130731
[2023-12-04T13:07:31.534+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-04T13:07:31.560+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
